### ISSUE TITLE
Option to skip/cancel/interrupt session restore

### ISSUE THREAD

**Issue by atulwy** [webyog/sqlyog-community#2068]
Original [issue 2069](https://code.google.com/p/sqlyog/issues/detail?id=2069) created by webyog on 2014-12-23T15:29:33.000Z:

User requested in Forums: http://forums.webyog.com/index.php?showtopic=7580

I think this is are reasonable request. ser is right that there are situations  (when waiting for connections to time-out when you know that one or more servers are not available, for instance) user is just left with an option to wait. Nothing else he can do.

**Comment by sibwy** [comment]
Another user also has asked for the cancellation of the restore process, when SQLyog launches untimely when a connection is not available any more.

The user request in the Forums >> http://forums.webyog.com/index.php?showtopic=7961

**Comment by sibwy** [comment]
An user has asked for cancelling the session restore as it hangs while restoring.

The user ticket in the helpscout >> https://secure.helpscout.net/conversation/145428789/138902/?folderId=115078

**Comment by sibwy** [comment]
Another user has asked for an option to cancel the session restore process.

The user request in the Forums >> http://forums.webyog.com/index.php?showtopic=9419

**Comment by Azzurvif** [comment]
Hello, since this thread comes in first with some google searches, I will share here a manual fix:
- Shut down SQLyog if it's stuck in the session loading
- Open the SQLyog folder in your %AddData%  at C:\Users\YourName\AppData\Roaming\SQLyog
- Rename or delete the file connrestore.db
I am working with a certain SQLyog revision, I don't know if it has been fixed in the last releases

**Comment by sibwy** [comment]
Another user has asked for this feature to cancel the session restore if a connection isn’t available while restoring the session on opening SQLyog and which will allow restoring the session again after ensuring the connection is established.

The user request can be found in HS >> # 166120

**Comment by mclaborn** [comment]
Yes, yes, a thousand times yes. This is my biggest single problem with SQLyog. 
I've also requested in that the restore be done in parallel, or to have configurable timeout values.

**Comment by sibwy** [comment]
A user has asked for an option to stop SQLyog from trying to make a connection outside the VPN if the connections connect only over a VPN. 

We have reports that timeout not always happens as expected when connection to MySQL is through VPN. Instead, the client (SQLyog) waits forever.

The user request can be found in HS >> #301330

**Comment by ArmSershant** [comment]
> Hello, since this thread comes in first with some google searches, I will share here a manual fix:
> 
> * Shut down SQLyog if it's stuck in the session loading
> * Open the SQLyog folder in your %AddData%  at C:\Users\YourName\AppData\Roaming\SQLyog
> * Rename or delete the file connrestore.db
>   I am working with a certain SQLyog revision, I don't know if it has been fixed in the last releases

Hello everybody still working, version of SQLYog 13.1.9
